docs/content/configuration/virtualserver-and-virtualserverroute-resources.md

```diff
@@ -95,6 +96,28 @@ basedOn: scheme
 |``basedOn`` | The attribute of a request that NGINX will evaluate to send a redirect. The allowed values are ``scheme`` (the scheme of the request) or ``x-forwarded-proto`` (the ``X-Forwarded-Proto`` header of the request). The default is ``scheme``. | ``string`` | No | ### VirtualServer.Policy |
 {{% /table %}}
 
+### VirtualServer.TLS.CertManager
+
+The certmanager field configures the provisioning of automated certificate resources using certmanager for a VirtualServer (Please note that ACME Issuers are not yet supported):
```

Comment:
If we're talking about the API objects we should capitalize Certificate. It helps disambiguate the API entity from the concept it represents.

The project consistently refers to itself as `cert-manager`. Staying consistent with how they refer to themselves should help users.